modules/EvseManager/Charger.cpp

```diff
@@ -45,7 +45,10 @@ Charger::Charger(const std::unique_ptr<board_support_ACIntf>& r_bsp, const std::
 }
 
 Charger::~Charger() {
-    r_bsp->call_pwm_F();
+    // FIXME: this cannot work since there is no guarantee that r_bsp is still around in shutdown
+    // TODO: do we need a pre-shutdown phase that does cleanup?
```

Comment:
exactly, we really need to think this throught, I guess like in the init, we should only shutdown if all modules successfully signaled, that they are shutdown although they might still receive signals